src/lib.rs

```diff
@@ -1121,6 +1121,33 @@ impl Element {
         .await
     }
 
+    /// Find and click an `<option>` child element by its index.
+    ///
+    /// This method clicks the first `<option>` element that is an `index`th child
+    /// (`option:nth-of-type(index+1)`). This will be the `index`th `<option>`
+    /// element if the current element is a `<select>`. If you use this method on
+    /// an `Element` that is _not_ a `<select>` (such as on a full `<form>`), it
+    /// may not do what you expect if there are multiple `<select>` elements
+    /// in the form, or if it there are stray `<option>` in the form.
+    ///
+    /// The indexing in this method is 0-based.
+    ///
+    /// It uses a CSS selector [nth-of-type] therefore it ignores other's
```

Comment:
I don't think you need this particular paragraph given the text that's now over :)